packages/react-scripts/config/polyfills.js

```diff
@@ -8,20 +8,20 @@
 // @remove-on-eject-end
 'use strict';
 
-if (typeof Promise === 'undefined') {
-  // Rejection tracking prevents a common issue where React gets into an
-  // inconsistent state due to an error, but it gets swallowed by a Promise,
-  // and the user has no idea what causes React's erratic future behavior.
-  require('promise/lib/rejection-tracking').enable();
-  window.Promise = require('promise/lib/es6-extensions.js');
-}
+//if (typeof Promise === 'undefined') {
```

Comment:
why not delete them?

Reply:
So i can remember what to add to shell in case there is something missing